### ISSUE TITLE
Missing Validation of Date Format if `StartDateTime` or `EndDateTime`  are missing

### ISSUE THREAD

**Issue by InputLabNet** [phax/validation#431]
## Description

Rule **BR-CO-19** doesn't enforce the date format for `StartDateTime` and `EndDateTime`  elements in `//ram:ApplicableHeaderTradeSettlement/ram:BillingSpecifiedPeriod`.

The rule **BR-29** can implicitly check that, but *only* if **both** `StartDateTime` and `EndDateTime` are present.

The same holds for `//ram:SpecifiedLineTradeSettlement/ram:BillingSpecifiedPeriod` (rules **BR-CO-20** and **BR-30**).

With that, the following example is reported as valid.

```xml
<ram:BillingSpecifiedPeriod>
  <ram:StartDateTime>
    <udt:DateTimeString format="date">first of May, 2024</udt:DateTimeString>
  </ram:StartDateTime>
</ram:BillingSpecifiedPeriod>
```

## Expected Behavior

* All date elements `udt:DateTimeString` should enforce the existence of `@format = '102'`.
* Validation should not depend on whether both date fields are present.
